android.pri

```diff
@@ -1,9 +1,33 @@
 include($$PWD/libs/qtandroidserialport/src/qtandroidserialport.pri)
 message("Adding Serial Java Classes")
 QT += androidextras
+
 ANDROID_PACKAGE_SOURCE_DIR = $$PWD/android
+android_source_dir_target.target = android_source_dir
+PRE_TARGETDEPS += $$android_source_dir_target.target
+QMAKE_EXTRA_TARGETS += android_source_dir_target
+
+exists($$PWD/custom/android) {
+    message("Merging $$PWD/custom/android/ -> $$PWD/android/")
+    ANDROID_PACKAGE_SOURCE_DIR = $$OUT_PWD/ANDROID_PACKAGE_SOURCE_DIR
+
+    android_source_dir_target.commands = $$QMAKE_MKDIR $$ANDROID_PACKAGE_SOURCE_DIR && \
```

Comment:
This allows, but doesn't force, the custom build to supply its own AndroidManifest.xml and java files.